USE-CASES/govtech-health-monitoring.md

```diff
@@ -0,0 +1,117 @@
+## Title: Public Health Monitoring
+
+### Submitter(s): 
+
+Jennifer Lin
+
+### Reviewer(s):
+
+Michael McCool, Michael Lagally
+
+### Tracker Issue ID:
+
+<please leave blank>
+
+### Category:
+
+<please leave blank>
+
+### Class: 
+
+<please leave blank>
+
+### Status: 
+
+<please leave blank>
+
+### Target Users
+
+Agencies, companies and other organizations in a Smart City with 
+significant foot traffic.
+
+### Motivation:
+
+A system to monitor the health of people in public places is useful to
+control the spread of infectious diseases.  In particular, we would like
+to identify individuals with temperatures outside the norm (i.e. running
+a fever) and then take appropriate action.
+
```

Comment:
It would be worth to add examples of the action, e.g. notfication via SMS or email, raising an immediate alert, closing a gate etc.
In some use cases the data is agregated from various locations (buildings, airports, border stations) for statistics and analytics.

Reply:
added (but I mention notification mechanisms later, so here I just say "notifications")